src/oc_erchef/apps/oc_chef_wm/src/chef_wm_actions_queue_monitoring.erl

```diff
@@ -306,9 +306,11 @@ handle_info(status_ping, #queue_monitor_state{
     Pid = spawn_link(
             fun () ->
                     Result =
-                      chef_wm_rabbitmq_management:check_current_queue_state(Vhost,
-                                                                            Queue,
-                                                                            Dropped),
+                      chef_wm_rabbitmq_management:check_current_queue_state(
+                        oc_chef_action_queue_config:get_rabbit_management_pool_name(),
```

Comment:
[minor] Perhaps we can throw the poolname in the queue_monitor_state in init() so we don't have to look it up every time.  This lookup is cheap so it isn't the end of the world to look it up, but since we already have the Vhost and Queue there it would be nice to be consistent (and some microseconds faster I suppose).